readme.txt

```diff
@@ -243,98 +243,69 @@ Our Cookie and Consent Banner can help you comply with GDPR. The European Union
 
 == Changelog ==
 
-= 9.0.2 =
+= 9.1 =
 
-* Release date: October 9, 2020
-
-**Enhancements**
-
-* Publicize: improve handling of URLs when generating Twitter threads.
-
-**Bug fixes**
-
-* Instagram Embeds: ensure that Instagram URLs with additional URL parameters can be embedded as well.
-* Media Extractor: prevent PHP notice for some URLs.
-* Publicize: avoid block editor errors when no Publicize connections are available.
-* Synchronization flow: avoid potential fatal errors when updating the plugin from the Dashboard > Updates screen.
-* Sharing: avoid PHP notices when a post object is invalid.
-* WordPress.com REST API: avoid errors when uploading images from a URL.
-
-= 9.0.1 =
-
-* Release date: October 6, 2020
-
-**Bug fixes**
-
-* General: remove a database optimization task that could potentially cause database issues.
-
-= 9.0 =
-
-* Release date: October 6, 2020
-* Release post: https://wp.me/p1moTy-rLy
+* Release date: November 3, 2020
+* Release post: https://wp.me/p1moTy-s0E
 
 **Major Enhancements**
 
-* Publicize: add the ability to publish the entire content of posts to Twitter as threads.
+* Story Block: this new block will allow you to add interactive stories to your posts, just like you would on your favorite Social Media platform.

```

Comment:
@Tug @aforcier Could you help me come up with a changelog entry for the Story block here, as well as a testing blurb for the testing list that will be sent to our Beta testers? See `to-test.md` below.

Thank you!

Reply:
Since it's going to be hidden from the picker, maybe we could specify that the creation is expected to happen from the mobile apps. In that case the changelog entry could be:

+* Story Block: this new block allows you to create and publish a story from the WordPress mobile apps.

I'll let @aforcier answer regarding the `to-test.md` as I haven't been involved in that part.

Reply:
👍 Done in 518c21ce56e583efc9c85f3da21be5487d326217

Reply:
Thanks @jeherve and @Tug !

I wonder if we could not to mention Stories at all this release, and instead announce it in the December release when we make it visible in the picker?

The reason is that the feature isn't released yet in the mobile apps (should be released to the beta channel in mid/late-November on Android and a bit later on iOS), so there's no reasonable way for anyone to try it (and no reasonable testing steps).

If we have to say something, we could maybe go for

> * Story Block: this new block will allow you to create and publish a story from the WordPress mobile apps in a future release.

But saying nothing would be preferable. WDYT @jeherve ?

Reply:
That works for me, we can wait until it's more publicly available.